_dprhtml/js/cped.js

```diff
@@ -2,11 +2,36 @@
 
 const getCPEDEntryAsObject = (velthiusWord) => {
   const entry = DPR_G.yt[velthiusWord] || [];
-  const [, grammar, definition, , inflectionGroup] = entry;
-  return { grammar, definition, inflectionGroup };
+  const [
+    ,
+    grammar,
+    definition,
+    ,
+    inflectionGroup,
+    inflectionInfo,
+    baseWord,
+    ,
+    stem,
+    regularity,
+  ] = entry;
+  return {
+    baseWord,
+    definition,
+    grammar,
+    inflectionGroup,
+    inflectionInfo,
+    isIrregular: regularity === "N",
+    stem,
+  };
 };
 
 const CPED = {
+  hasEntry(velthiusWord) {
+    return this.getDefinition(velthiusWord) !== undefined;
+  },
+  getBaseWord(velthiusWord) {
+    return getCPEDEntryAsObject(velthiusWord).baseWord;
+  },
```

Comment:
Just as a note, this API is currently providing very low-level pieces of information. My hope is that as more of the CPED logic is taken out of the DPR, we can create higher-level abstractions that will eventually be replaced by the DPD.